### ISSUE TITLE
CryptoCurrencies table 'Monero' payment link field is messed up.

### ISSUE THREAD

**Issue by mesnevi** [WeblateOrg/website#35]
Greetings, 

I've been browsing [Weblate.org Donation page](https://weblate.org/en/donate/) and noticed, that **Monero** payment link field is messed up (please see screenshot attached):

![weblate-crypto-donate-table](https://user-images.githubusercontent.com/1187496/45873775-e424d600-bd9b-11e8-8b10-5f27b2ebd879.png)

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.

**Comment by mesnevi** [comment]
Wow, what a speed! I was going to fix it myself but was like a couple of minutes behind. 

I am very impressed with very detailed though not too wordy Weblate documentation and with website layout itself (I've never seen feature description pointing to the Docs articles, it's very simple and cute idea!). I was impressed while reading your communication with F-Droid guys, and I'm doubly impressed with the development speed and quality. I need to take it as a role model, Sir (*with my hat off*) Thank you for the decent work!

**Comment by nijel** [comment]
Well you were mostly lucky that I was online and working on Weblate at that time ;-).
